client/pages/details.jsx

```diff
@@ -110,14 +127,27 @@ export default class Details extends React.Component {
     return (
     <div className='container'>
       <div style={{ backgroundImage: `url(https://images.igdb.com/igdb/image/upload/t_screenshot_huge/${this.state.backgroundImage})`, backgroundRepeat: 'no-repeat' }} className='row min-height-background-image background-size'>
-        <div className='col'>
+        <div className='col-12 col-md-6 position-rel'>
           <img src={`https://images.igdb.com/igdb/image/upload/t_cover_small_2x/${this.state.gameInfo[0].cover.image_id}.jpg`} id="game-logo"></img>
-        </div>
-        <div className='col'>
-            <h4 id="game-title" className='color-text-white font-lig margin-right-small'>{`${name} (${dateTest})`}</h4>
-        </div>
-        <div className='col'>
-            <Badge id="rating" bg="info">{rating}</Badge>
+          <h4 id="game-title" className='color-text-white font-lig fs-4 margin-right-small'>{`${name} (${dateTest})`}</h4>
+          <Badge id="rating" bg="info">{rating}</Badge>
+          <div className='star-position'>
+            {this.state.avgRating !== undefined
+              ? this.state.avgRating !== undefined && (
+                new Array(5).fill().map((star, index) => {
+                  index += 1;
+                  return (
+                    <i key={index} className={index <= this.state.avgRating.avg ? 'fa-solid fa-star gold star-size' : 'fa-regular fa-star heading-star star-size'}></i>
+                  );
+                }))
+              : new Array(5).fill().map((star, index) => {
+                index += 1;
+                return (
+                  <i key={index} className="fa-regular fa-star heading-star star-size"></i>
+                );
+              })
+           }
```

Comment:
I recommend starting your average rating at 0 instead of undefined so you don't need as much conditional logic.